src/scripts/angry-tock.js

```diff
@@ -112,64 +117,61 @@ const shout = async ({ calm = false } = {}) => {
  */
 const isAngryTockDay = (now) => {
   const d = now || m();
-  return d.format("dddd") === "Monday" && !holidays.isAHoliday(d.toDate());
+  return d.dayOfWeek === 1 && !holidays.isAHoliday(dateFromZonedDateTime(m()));
```

Comment:
😭  `.dayOfWeek === 1` is my least favorite part of native Date class

Reply:
Yeah, it bit me *several* times. I finally gave up trying to keep it straight in my head and created an enum so I can do `.dayOfWeek === DAYS.Monday` instead.